### DISCUSSION TITLE
Hide Column on Kanban Board

### DISCUSSION THREAD

**Discussion by vshcherb** [community/community#6477] [Projects and Issues] [answered]
It's nice to have same representation of the project on the board especially nice to have different Kanban Boards.

We would like to implement series of Kanban boards:

Idea -> Design -> Development -> Deployment (each of them with separate  progress columns).

It's all possible to do now though all Board displays all columns which clutters the view a lot. 

**Suggestion**: It would be nice to hide certain columns for each Kanban board

**Comment by meloracybul** [comment]
Just coming by to upvote and add some thoughts. I want two Kanban tabs that splits between Dev and Quality so we can 1) get rid of horizontal scrolling and 2) focus on what each team needs while retaining the seamless workflow across the entire path.

**Comment by 2percentsilk** [comment, marked as answer]
You can now hide columns in the board view by updating your filter!
See our previous [changelog](https://github.blog/changelog/2022-09-02-the-new-github-issues-september-2nd-update/)
